EIPS/eip-5792.md

```diff
@@ -55,13 +55,14 @@ The wallet:
 ```typescript
 type SendCallsParams = {
   version: string;
-  chainId: `0x${string}`; // Hex chain id
   from: `0x${string}`;
   calls: {
     to?: `0x${string}` | undefined;
     data?: `0x${string}` | undefined;
     value?: `0x${string}` | undefined; // Hex value
+    chainId?: `0x${string}` | undefined; // Hex chain id
   }[];
+  parallel?: boolean;
```

Comment:
seems like leftover? i think we should remove this. lmk if we're not aligned here though